reference/binary_model/custom_compatibility.rst

```diff
@@ -0,0 +1,166 @@
+Customizing the binary compatibility
+====================================
+
+The default binary compatibility requires an almost exact match of settings and options, and a versioned match
+of dependencies versions, as explained in the :ref:`previous section about dependencies<reference_binary_model_dependencies>`.
+
+In summary, the required binaries ``package_id`` when installing dependencies should match by default:
+
+- All the settings in the ``package_id`` except ``compiler.cppstd`` should match exactly the ones provided in the input profiles, including the compiler version. So ``compiler.version=9`` is different than ``compiler.version=9.1``.
+- The default behavior will assume binary compatibility among different ``compiler.cppstd`` values for C++ packages, being able to fall back to other values rather the specified in the input profiles, if the ``cppstd`` required by the input profile does not exist. This is controlled by the ``compatibility.py`` plugin, that can be customized by users.
```

Comment:
```suggestion
- The default behavior will assume binary compatibility among different ``compiler.cppstd`` values for C++ packages, being able to fall back to other values rather than the one specified in the input profiles, if the ``cppstd`` required by the input profile does not exist. This is controlled by the ``compatibility.py`` plugin, that can be customized by users.
```